### ISSUE TITLE
A11Y: Video should only set [title], not [alt]

### ISSUE THREAD

**Issue by dreamofabear** [GoogleForCreators/web-stories-wp#4978]
## Bug Description

There are two text fields under Design > Accessibility for videos: "Title" and "Assistive Text".

<img width="278" alt="Screen Shot 2020-10-21 at 12 26 59 PM" src="https://user-images.githubusercontent.com/20525523/96749381-bd7d6300-1398-11eb-9269-495d5d9655fa.png">

## Expected Behaviour

There should only be one field ("Assistive Text") which sets `amp-video[title]`. Note that this is intentionally different than images which set `amp-img[alt]`. 

> We recommend that you take full advantage of semantic HTML to build up your story. However, some tools designed for the social media use case may instead export a story such that each slide is represented as a video file that bakes in all the text into the video. In this case it is recommended to add the precise text displayed inside of the video as a “title” attribute on the amp-video element. Again, only do this if you absolutely cannot use semantic markup in your Stories generation.

https://blog.amp.dev/2020/02/12/seo-for-amp-stories/

## Steps to Reproduce

N/A

## Screenshots

See above.

## Additional Context

- Plugin Version:
- Operating System:
- Browser:

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

**Comment by swissspidy** [comment]
Duplicate of #4708

See also https://github.com/ampproject/amphtml/issues/30415
